applications/crossbar/doc/basics.md

```diff
@@ -1,340 +1,218 @@
+        "is_reseller": false,
+        "language": "en-us",
+        "music_on_hold": {},
+        "name": "child account",
+        "preflow": {},
+        "realm": "aeac33.sip.2600hz.com",
+        "reseller_id": "undefined",
+        "ringtones": {},
+        "superduper_admin": false,
+        "timezone": "America/Los_Angeles",
+        "wnm_allow_additions": false
+    },
     "request_id": "{REQUEST_ID}",
     "revision": "{REVISION}",
-    "start_key": "g2wAAAACbgUAb0ZX1A5oAWpq",
     "status": "success"
 }
 ```
 
-Observe now that `start_key` is the requested `start_key` and `next_start_key` points to the start of the next page of results.
-
-!!! note
-    If `next_start_key` is missing from the response envelope, the response represents the last page of results.
-
-You can also choose to receive pages in bigger or smaller increments by specifying `page_size` on the request. Do take care, as the `next_start_key` will probably vary if you use the same `start_key` but differing `page_size` values.
-
-### Setting Page Size
-
-By default, API requests have a page size of 50 results. This value is customizable by system administrator in the `crossbar.maximum_range` system config setting.
+As you can see, this account endpoint will contain configurations related to your account. You can get more information about this endpoint from the Accounts documentation section.
 
-For individual API request, you can also include a `page_size` query string parameter. For example: `http://{SERVER}:8000/v2/{API_URL}?page_size=25`.
+## KAZOO URI Structure
 
-### Setting sorting order
+When you make a request to KAZOO, the majority of requests you’ll want to make act upon a couple of important identifiers.
 
-By default, Crossbar returns the results in descending order. To get results in ascending order either set `ascending=true` (Kazoo 4.2+ only) or `descending=false` in the request query string.
+Let’s break down the following URI structure:
 
-!!! note
-    The field used to sort the individual API results depends on the internal implementation of the API endpoint and is not controllable by the client.
+`/{VERSION}/accounts/{ACCOUNT_ID}/resources/{RESOURCE_ID}`
 
-### Disabling Pagination
+The components of this URI are:
 
-If you want to disable pagination for a request, simply include `paginate=false` on the query string.
+1. `{VERSION}`
+    - The version of the API endpoint, currently `v2` for all supported APIs found in this documentation.
 
-#### Protecting from (un)intentional abuse
+1. `/accounts/{ACCOUNT_ID}`
+    - This is your KAZOO account ID, a 32-character hex-encoded identifier.
 
-Since pagination can be turned off by a client-supplied query string parameter, it is important that KAZOO still protect itself from overly large datasets being loaded. Examples seen include large CDR listings, call recording listings, and ledger listings.
+1. `/{RESOURCE}/{RESOURCE_ID}`
+    - A resource, such as `devices`, `users`, `callflows`, et cetera.
```

Comment:
So you should probably not use `resource` here as in KAZOO a `resource` represents upstream carrier configs.

We generically call them `collection` and `entity`, so `accounts` is the collection, `{ACCOUNT_ID}` is the entity.